app/services/generate_single_page_list_params_service.rb

```diff
@@ -0,0 +1,16 @@
+module GenerateSinglePageListParamsService
+  def generate_single_page_list_params(content_item)
```

Comment:
I think we could maybe put a comment here, with a link to the code in email alert frontend.

Reply:
OK done, this is my comment:

```  # This is equivalent to code introduced into email alert frontend:
  # https://github.com/alphagov/email-alert-frontend/blob/main/app/services/subscriber_list_params/generate_single_page_list_params_service.rb
  # to handle subscriber lists with reverse links.
  # It will only be needed in here temporarily while we migrate specialist topics
```

Happy to finesse in any way, I'm not sure about language here